msg/BoundingBox2D.msg

```diff
@@ -1,9 +1,9 @@
 # Note: This message plans to move to geometry_msgs in the final implementation.
 
-# The x/y position and (optional) rotation of the bounding box center.
-geometry_msgs/Pose2D pose
+# The 2D position (in pixels) and orientation of the bounding box origin.
+geometry_msgs/Pose2D origin
```

Comment:
Honest question: Is it common to have rotated bounding boxes in 2D classification? I've only seen axis-aligned bounding boxes so far. If so, we should replace the Pose2D with a "Point2D" (which doesn't exist yet), preferably with uint32 coordinates instead of float. If (almost) nobody uses the rotation, many people won't implement it correctly.